Packs/Phishing/Playbooks/Phishing_Investigation_-_Generic_v2_-_6_0.yml

```diff
@@ -179,19 +179,21 @@ tasks:
       - "53"
       - "85"
     scriptarguments:
+      append: {}
       key:
         simple: ReporterAddress
+      stringify: {}
       value:
         complex:
-          root: incident
-          accessor: labels.Email/from
+          root: ExtractedIndicators.Email
+          accessor: '[0]'
```

Comment:
is the [0] intentional?

Reply:
Yes. The extracted email address will be always the first item.